osu.Game/Beatmaps/Drawables/UpdateableBeatmapBackgroundSprite.cs

```diff
@@ -46,17 +46,21 @@ protected override DelayedLoadWrapper CreateDelayedLoadWrapper(Drawable content,
         protected override Drawable CreateDrawable(BeatmapInfo model)
         {
             Drawable drawable;
+            BeatmapInfo localInfo;
 
-            var localBeatmap = beatmaps.GetWorkingBeatmap(model);
+            if (model != null && model.ID == 0)
+                localInfo = beatmaps.QueryBeatmap(p => p.OnlineBeatmapID == model.OnlineBeatmapID);
```

Comment:
this is going to catastrophically fail if `model.OnlineBeatmapID` is null. it'll probably return a random beatmap.